### DISCUSSION TITLE
Martin's anwers

### DISCUSSION THREAD

**Discussion by rchastain** [mse-org/mseide-msegui#74] [Show and tell]
Hello friends. I had a big text file on my HD (don't remember where I got it) with all messages from Sourceforge mailing list. I extracted all Martin's answers and saved them to separate files (one discussion, one file). It is [here](https://codeberg.org/rchastain/msedoc/src/branch/master/text).

The original file is very big (more than 200 000 lines). That's why I decided to keep only Martin's messages.

By the way, I also cleaned my personal [website](https://msegui.net/). If you want to take a look...

**Comment by rchastain** [comment]
There are 1135 files!

**Comment by fredvs** [comment]
WoooW!
Martin has very good friends!

**Comment by rchastain** [comment]
Yes! I hope you will also like what I have just made from a Martin's anwer:

- [Tutorial: Properties face and frame of twidget](https://msegui.net/tutoriels/twidget-frame-face/tutoriel.html)

**Reply by fredvs** [reply]
Super clear and easy to understand.
Great addition.
Thanks Roland.

**Reply by fredvs** [reply]
There is also the "custom frame", with rounded corner and shadows: https://github.com/mse-org/mseuniverse/tree/main/skins/fredvs

**Reply by rchastain** [reply]
Fred, please, create a new discussion.

Here the topic is: what we could do with Martin's answers.
